src/transforms/parameterTransforms.ts

```diff
@@ -151,11 +165,14 @@ const isClientImplementation = (parameter: Parameter) =>
   parameter.implementation === ImplementationLocation.Client;
 
 function getParameterLocation(parameter: Parameter): ParameterLocation {
-  if (!parameter.protocol.http) {
+  const originalLocaltion = (parameter.protocol.http || { in: undefined }).in;
```

Comment:
Nit: The "in: undefined" isn't necessary since "in" will already be undefined by being absent.
```suggestion
  const originalLocaltion = (parameter.protocol.http || {}).in;
```

Side-note: I want optional chaining from TypeScript 3.7...

Reply:
Thanks for the suggestion, I'm upgrading to 3.7 and using optional chaining :smile: